src/main/java/frc/robot/commands/intake/DefaultIntake.java

```diff
@@ -0,0 +1,57 @@
+package frc.robot.commands.intake;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Conveyor;
+import frc.robot.subsystems.Conveyor.ConveyorState;
+import frc.robot.subsystems.Intake;
+
+/**
+ * Default command for Intake.
+ */
+public class DefaultIntake extends CommandBase {
+
+    private final Intake intake;
+    private final Conveyor conveyor;
+
+    /**
+     * Default command for Intake.
+     *
+     * @param intake The intake
+     * @param conveyor The conveyor
+     */
+    public DefaultIntake(Intake intake, Conveyor conveyor) {
+        addRequirements(intake, conveyor);
+        this.intake = intake;
+        this.conveyor = conveyor;
+    }
+
+    @Override
+    public void initialize() {
+    }
+
+    @Override
+    public void execute() {
+        switch (intake.getState()) {
+            case DOWN:
+                if (conveyor.getFront() != ConveyorState.NONE
+                    && conveyor.getBack() != ConveyorState.NONE) {
+                    intake.setIntakeMotorPower(-1.0);
```

Comment:
it shouldent run backwards all the time, that is a waist of power. have it run backwards for ~1s and then stop the motor